src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -0,0 +1,51 @@
+package roomescape.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicLong;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import roomescape.entity.Reservation;
+import roomescape.dto.ReservationRequestDto;
+
+@Controller
+public class ReservationController {
+
+    private final List<Reservation> reservations = new ArrayList<>();
+    private final AtomicLong index = new AtomicLong(1);
+
+    @GetMapping("/admin")
+    public String admin() {
+        return "/admin/index";
+    }
+
+    @GetMapping("/admin/reservation")
+    public String reservation() {
+        return "/admin/reservation-legacy";
+    }
+
+    @GetMapping("/reservations")
+    public ResponseEntity<List<Reservation>> findAllReservations() {
+        return ResponseEntity.ok(reservations);
+    }
+
+    @PostMapping("/reservations")
+    public ResponseEntity<Reservation> addReservation(@RequestBody ReservationRequestDto reservationRequestDto) {
+        Reservation reservation = reservationRequestDto.toEntity(index.getAndIncrement());
+        reservations.add(reservation);
+        return ResponseEntity.ok(reservation);
+    }
+
+    @DeleteMapping("/reservations/{id}")
+    public ResponseEntity<Void> deleteReservation(@PathVariable Long id) {
```

Comment:
```suggestion
    public ResponseEntity<Void> deleteReservation(@PathVariable long id) {
```

Long / long는 무엇이 다른가요? 여기서는 무엇을 사용하는게 좋을까요?

Reply:
> Long / long는 무엇이 다른가요? 여기서는 무엇을 사용하는게 좋을까요?

Long과 long의 차이를 생각해 봤을 때 가장 먼저 떠오르는건 null이 가능한가? 인것 같습니다. 이 관점에서 남겨주신 의견을 생각해보면, 지금 단계에선 `Long`으로 선언된 값들이 null이 될 여지가 없다고 느껴지기에 long을 사용해도 충분하다는 생각이 들었습니다!